README.md

```diff
@@ -30,7 +30,13 @@
 
 # 📖 React Group Project <a name="about-project"></a>
 
+<<<<<<< HEAD
```

Comment:
Please clean up your readme file by removing these lines as it will make your readme file more professional.